script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -459,32 +459,32 @@ Instead, she's focusing her attention on something completely different.
 [sha:9ce9478443ada8539868dbc23ff5fc1b70383084]{
 -- Page 406, Offset 9656.
 -- 「さ、どうしようか志貴。
-"So what's the plan, Shiki?
+"So, what's the plan, Shiki?
 }
 [sha:16371981eab054519baf65e0a07eda6abf9d7a35]{
 -- Page 406, Offset 9657.
 -- 見ての通り、ヴローヴはあの状態よ。わたしを探し出そうとする理性すらなかった只の殺戮者」
-As you can see, this is Vlov's true nature. He's a mass murderer that had no intention of finding me in the first place."
+As you can see, this is Vlov's true nature. He's some crazed mass murderer that's too far gone to even track me down."
```

Comment:
Minor point, but this suggests that he _couldn't_ rather than that _he lacks the sanity to want to_